docs/ops-recipe/harborPodPhaseStatus.md

```diff
@@ -0,0 +1,86 @@
+---
+
+title: "harborPodPhaseStatus"
+
+---
+
+This alert fires because one or more Harbor pods are in a failing, pending or unknown [phase](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/). This could occur during deployment or at any point during Harbors lifecycle. 
+
+
+**Harbor exporter**
+
+This is a specific case for when a disruption occurs with the `harbor-exporter` pod, which might lead to issues with exporting metrics to Prometheus, and connecting with Postgres.
+
+Likely you wont see this in the terminal and instead will receive the notification through prometheus. Restart the harbor-exporter pod and check the logs.
+
+```
+
+kubectl -n harbor-cluster delete pod harbor-exporter
+
+kubectl -n harbor-cluster logs harbor-exporter
+
+```
+
+The error is likely to do with the connection to the exporter and other components being down. Try restarting the service. 
+
+```
+
+kubectl -n harbor-cluster delete service harbor-exporter
```

Comment:
Are you sure? This should change nothing, only refresh service endpoints. Should this be restart a Deployment?